test/fail_compilation/imports/test21651a.d

```diff
@@ -0,0 +1,5 @@
+module imports.test21651a;
+
+// both statements should be error
```

Comment:
The contents of this file should be moved to `fail_compilation/test21651.d`. That way we use only 2 files.

Reply:
Done.